pkg/cmd/list/root.go

```diff
@@ -0,0 +1,48 @@
+package list
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/kind/pkg/cluster"
+)
+
+var (
+	// Flags
+	buildName string
+)
+
+var ListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List idp clusters",
+	Long:  ``,
+	RunE:  delete,
+}
+
+func init() {
+	ListCmd.PersistentFlags().StringVar(&buildName, "buildName", "localdev", "Name of the kind cluster to be deleted.")
```

Comment:
Should be `build-name` instead of `buildName` because we use kebab case everywhere else.